### DISCUSSION TITLE
AWX mail module not working

### DISCUSSION THREAD

**Discussion by rakeshsujan** [community/community#139055] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Ansible mail module not working in AWX but working in CLI

**Comment by LiteBrite82** [comment]
Hi @rakeshsujan ,

Thanks for being a part of the GitHub Community, we're glad you're here!

It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
